proto/storage/base_image.proto

```diff
@@ -20,8 +19,8 @@ message BaseImage {
 }
 
 message BaseImageLayer {
-  string id = 1; // @gotags: sql:"pk"
-  string base_image_id = 2; // @gotags: sql:"fk(BaseImage:id),no-fk-constraint"
+  string id = 1; // @gotags: sql:"pk,type(uuid)"
+  string base_image_id = 2; // @gotags: sql:"fk(BaseImage:id)"
   string layer_digest = 3; // @gotags: search:"Base Image Layer Digest,hidden"
```

Comment:
Can we make unique(base_image_id, layer_digest)? Also, do you know if DELETE CASCADE will apply to this base image layer when base image is deleted?

Reply:
I do believe DELETE CASCADE will apply to this base image layer, please see the PR description "How I validated my change"

Reply:
Will add `unique(base_image_id, layer_digest)`

Reply:
> Will add `unique(base_image_id, layer_digest)`

Added